### PR TITLE
added element chart config options

### PR DESCRIPTION
**Pull request by Engelskri3ger**
Added unit tests like they were in other classes.
I think it would be great to use this wrapper with https://github.com/Ejdamm/chart.js-php to get a complete php package for chartjs.

### CODE REVIEW

File: src/Options/Elements/Line.php [hunk 1]
```diff
@@ -0,0 +1,326 @@
+<?php
+
+namespace Halfpastfour\PHPChartJS\Options\Elements;
+
+
```
Comment (halfpastfouram) [inline src/Options/Elements/Line.php]: Please bring this back to one empty line.

Comment (Engelskri3ger) [reply src/Options/Elements/Line.php]: done

**Comment by halfpastfouram** [comment]
Looks great! Thanks.

**Comment by SanderMuller** [comment]
I get the following unit test failure since this PR was merged:
There was 1 failure:

```
Test\Options\Elements\LineTest::testJsonSerialize Failed asserting that 
Array &0 (
    'tension' => 0.2
    'backgroundColor' => 'backgroundColor'
    'borderWidth' => 2
    'borderColor' => 'borderColor'
    'borderCapStyle' => 'borderCapStyle'
    'borderDash' => Array &1 (
        0 => 5
        1 => 6
    )
    'borderDashOffset' => 0.1
    'borderJoinStyle' => 'borderJoinStyle'
    'fill' => true
) is identical to Array &0 (
    'tension' => 0.2
    'backgroundColor' => 'backgroundColor'
    'borderWidth' => 2
    'borderColor' => 'borderColor'
    'borderCapStyle' => 'borderCapStyle'
    'borderDash' => Array &1 (
        0 => 5
        1 => 6
    )
    'borderDashOffset' => 0.1
    'borderJoinStyle' => 'borderJoinStyle'
    'capBezierPoints' => true
    'fill' => true
    'stepped' => true
).

**Pull request merged**